src/helpers/Experiments.spec.jest.js

```diff
@@ -0,0 +1,86 @@
+const Experiments = require('./Experiments');
```

Comment:
In this PR, in addition to adding the Jest version of the test file - we should remove the Karma version.  While we're working through the conversion, it probably makes sense to also change the `test` script in package.json to run both Karma and Jest